learn/app-development/widgets/datalive/form/form-field-validator.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Validator Function"
+id: ""
+sidebar_label: "Field Validations"
+---
+---
+
+Apply validations to a single field and multiple fields using the specified methods. These methods on the form field can be used to set sync validations on a field. These validations can include default validators like required, min value, max value, or add more customizations to the basic validations.
+
+There are three ways to apply Validations on a form field based on your requirements; they are:
+
+- `setValidators`
+- `observeOn`
+- `setAsyncValidators`
+
+## setValidators
+
+For setting default validations for Form fields, you can use the `setValidator()` method. This method accepts an array of:
+
+- Objects for default validators
```

Comment:
Instead say that WaveMaker ship a bunch of validators and these can be referenced and used in the setValidator method

Reply:
Updated the doc.